webapp/src/components/App/Singlestudy/explore/Modelization/Map/style.ts

```diff
@@ -0,0 +1,87 @@
+import { styled, Box, Chip } from "@mui/material";
+import mapbackground from "../../../../../../assets/mapbackground.png";
+
```

Comment:
We use `sx` when the styles are used only one time.
`styled` when they are used multiple times.